normalize_repos/constants.py

```diff
@@ -0,0 +1,55 @@
+# Standard library
```

Comment:
I think this file should only be for issues related to branch protection, maybe call it `branch_protections.py`

Reply:
Okay. Will update it